### PR TITLE
Placeholders

### PR DESCRIPTION
**Pull request by RoCKnD79**
Created the UI of the placeholders where a user will be able to display the pictures relative to their favorite songs and favorite artists.

In addition to that I moved the navigation drawer button from the editprofile activity to the profile activity which made more sense.

image

### CODE REVIEW

File: app/src/main/java/com/github/orkest/View/NavigationBar.kt [hunk 1]
```diff
@@ -79,7 +82,9 @@ class NavigationBar {
                     composable("HomePage") { FeedActivity() }
                     composable("SearchPage") { SearchUserView.SearchUi(viewModel = viewModel) }
                     composable("PlaylistPage") { Text(text = "Playlist tab") }
-                    composable("ProfilePage") { ProfileTopInterface(viewModel = ProfileViewModel(user = currentUser)) }
+                    composable("ProfilePage") {
+                            ProfileActivityScreen(ProfileActivity(), viewModel = ProfileViewModel("JohnDoe"))
+                    }
                 }
```
Comment (saraanej) [inline app/src/main/java/com/github/orkest/View/NavigationBar.kt:88]: Why did you change the ProfileViewModel parameter ? Do your tests not pass without that change ?

File: app/src/main/java/com/github/orkest/View/profile/ProfileActivity.kt [hunk 2]
```diff
@@ -22,28 +49,139 @@ class ProfileActivity() : ComponentActivity() {
...
+    ProfileTopInterface(viewModel = viewModel, scaffoldState, coroutineScope)
+}
+
+@Composable
+fun mainBody() {
+    Divider(modifier = Modifier.padding(vertical = 10.dp))
+    favoriteSongs()
+    Divider(modifier = Modifier.padding(vertical = 10.dp))
+    favoriteArtists()
+}
```
Comment (saraanej) [inline app/src/main/java/com/github/orkest/View/profile/ProfileActivity.kt:120]: Maybe putting these values into variables would be better if future changes are needed

File: app/src/androidTest/java/com/github/orkest/View/profile/ProfileUITest.kt [hunk 3]
```diff
@@ -43,7 +43,9 @@ class ProfileUITest {
         )
 
         composeTestRule.setContent {
-            OrkestTheme { topProfile(viewModel = viewModel) }
+            ProfileActivitySetting {
+                ProfileActivityScreen(ProfileActivity(), viewModel = viewModel)
+            }
         }
     }
```
Comment (saraanej) [inline app/src/androidTest/java/com/github/orkest/View/profile/ProfileUITest.kt:50]: This line of code will have to be added in each of your test functions to avoid merge conflicts

File: app/src/androidTest/java/com/github/orkest/View/profile/ProfileUITest.kt [hunk 4]
```diff
@@ -56,6 +58,31 @@ class ProfileUITest {
...
+        composeTestRule.onNodeWithText("Favorite Songs").assertIsDisplayed()
+        composeTestRule.onNodeWithText("Favorite Artists").assertIsDisplayed()
+    }
+
+    @Test
+    fun navigationDrawerComponentsDisplayOnScreen() {
+        // drawer elements displayed when opening the drawer
+        composeTestRule.onNodeWithContentDescription("Drawer Icon").performClick()
+        composeTestRule.onNodeWithText("Notifications").assertIsDisplayed()
+        composeTestRule.onNodeWithText("Privacy").assertIsDisplayed()
```
Comment (saraanej) [inline app/src/androidTest/java/com/github/orkest/View/profile/ProfileUITest.kt:71]: Nice tests

File: app/src/main/java/com/github/orkest/View/profile/ProfileActivity.kt [hunk 5]
```diff
@@ -22,28 +49,139 @@ class ProfileActivity() : ComponentActivity() {
...
+        }
+
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ProfileActivityScreen(activity: ComponentActivity, viewModel: ProfileViewModel) {
+    val scaffoldState = rememberScaffoldState()
+    val coroutineScope = rememberCoroutineScope()
```
Comment (saraanej) [inline app/src/main/java/com/github/orkest/View/profile/ProfileActivity.kt:77]: Maybe add comments to understand what these two variables represent

**Review by saraanej — APPROVED** [review]

File: app/src/androidTest/java/com/github/orkest/View/profile/ProfileUITest.kt (hunk 4 above)
Comment (RoCKnD79) [reply app/src/androidTest/java/com/github/orkest/View/profile/ProfileUITest.kt]: toi nice test

File: app/src/main/java/com/github/orkest/View/profile/ProfileActivity.kt [hunk 6]
```diff
@@ -22,28 +49,139 @@ class ProfileActivity() : ComponentActivity() {
...
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    topProfile(viewModel = viewModel, scaffoldState, coroutineScope)
+                }
+                mainBody()
+            }
+        },
+        drawerGesturesEnabled = true
```
[the remark this answers is not in the document]
Comment (RoCKnD79) [reply app/src/main/java/com/github/orkest/View/profile/ProfileActivity.kt:105]: thank you
